packages/@aws-cdk/aws-rds/lib/props.ts

```diff
@@ -446,6 +446,12 @@ interface CommonRotationUserOptions {
  * Options to add the multi user rotation
  */
 export interface RotationSingleUserOptions extends CommonRotationUserOptions {
+  /**
+   * The secret to rotate, must be already attached!
+   *
+   * @default - master secret of the instance
```

Comment:
This is used both for Instance and Cluster, right?

```suggestion
   * @default - master secret of the instance/cluster
```